pkgs/development/python-modules/sip/default.nix

```diff
@@ -15,19 +16,20 @@
 
 buildPythonPackage rec {
   pname = "sip";
-  version = "6.8.3";
+  version = "6.8.6";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-iIVHsBi7JMNq3tUZ6T0+UT1MaqC6VbfMGv+9Rc8Qdiw=";
+    hash = "sha256-f8lZ5I5uxdWvi9Am9p9eJNCLPLirs0IXb1q4AwzAfXo=";
   };
 
   nativeBuildInputs = [ setuptools ];
 
   propagatedBuildInputs = [
     packaging
     setuptools
+    setuptools-scm
```

Comment:
Why is this propagated?

Reply:
Followed the same setuptools pattern assuming it had to be propagated along setuptools

Reply:
It's not in `dependencies`: https://github.com/Python-SIP/sip/blob/main/pyproject.toml#L15